sys-apps/cpuid/cpuid-20180519.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Linux tool to dump x86 CPUID information about the CPUs"
+HOMEPAGE="http://www.etallen.com/cpuid.html"
+SRC_URI="http://www.etallen.com/${PN}/${P}.src.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
```

Comment:
Would both of these be a BDEPEND instead? Could you check (minor issue, rest looks good)

Reply:
Yeah, perl should go into BDEPEND. But FWICS, gzip should stay in DEPEND. Updated PR.